### ISSUE TITLE
Crash due to Royal Red's fire breath

### ISSUE THREAD

**Issue by TeamCapumon** [WolfShotz/Wyrmroost#137]
**Logs:**
[crash-2020-11-10_02.03.09-client.txt](https://github.com/WolfShotz/Wyrmroost/files/5516461/crash-2020-11-10_02.03.09-client.txt)

**Steps to Reproduce:**
 1. Ride an alpine dragon over long stretches of chunks on your admittedly pretty slow computer
 2. See a nearby sleeping royal red dragon (not tamed)
 3. Run away because that's the same dragon that just tried to kill you like 2 minutes ago
 4. Accidentally land on a tree while trying to turn around to see if it's following you
 5. Game crash
 6. Restart game, game crashes again, rip

**Description of issue:**
Basically exactly what that said. I'm not a programmer so I don't know what the issue is, but I do see "fire breath" and "wyrmroost" so that narrows it down a bit. Tfw a hostile enemy dragon kills you so hard it kills your entire minecraft world

I think all the mods I have installed are included at the bottom of the crash report yes? if you need any more information please let me know and I'll do my best to provide it!

**Comment by TeamCapumon** [comment]
Oh funny, this got closed right when I was about to say it fixed for me, LOL!
